include/unifex/at_coroutine_exit.hpp

```diff
@@ -20,6 +20,8 @@
 #include <unifex/await_transform.hpp>
 #include <unifex/continuations.hpp>
 #include <unifex/stop_token_concepts.hpp>
+#include <unifex/unstoppable_token.hpp>
+#include <unifex/get_stop_token.hpp>
```

Comment:
Were these necessary for this change or simply missing?

Reply:
Without these, the msvc build fails.